docs/src/adding_models_for_general_use.md

```diff
@@ -543,7 +543,18 @@ MLJBase.package_uuid(::Type{<:DecisionTreeClassifier}) = "7806a523-6efd-50cb-b5f
 MLJBase.package_url(::Type{<:DecisionTreeClassifier}) = "https://github.com/bensadeghi/DecisionTree.jl"
 MLJBase.is_pure_julia(::Type{<:DecisionTreeClassifier}) = true
 ```
-
+Alternatively these traits can also be declared using `MLJBase.metadata_pkg` and `MLJBase.metadata_model` helper functions as:
+```julia
+MLJBase.metadata_pkg(DecisionTreeClassifier,name="DecisionTree",
+                     uuid="7806a523-6efd-50cb-b5f6-3fa6f1930dbb",
+                     url="https://github.com/bensadeghi/DecisionTree.jl",
+                     julia=true)   
+            
+MLJBase.metadata_model(DecisionTreeClassifier,
+                        input=MLJBase.Table(MLJBase.Continuous),
+                        target=AbstractVector{<:MLJBase.Finite},
+                        output=Unknown,path="MLJModels.DecisionTree_.DecisionTreeClassifier")
+ ```
 You can test all your declarations of traits by calling `MLJBase.info_dict(SomeModel)`.
```

Comment:
Drop the `output` kwarg. On the one hand `Unknown` is the default. On the other, `output_scitype` is not actually a trait for supervised models - only supervised ones. It seems the macro goes ahead and defines this trait regardless (to avoid separate macros ? @tlienart ) but this does not appear to do any harm. 

@tlienart Can you double check this PR (after above change) and merge if you're happy?

Reply:
Looks good to me, thank you @OkonSamuel!